appium/tests/helpers/ElementHelper.ts

```diff
@@ -56,6 +56,19 @@ class ElementHelper {
     await element.click();
   }
 
+  static deleteText = async (element: WebdriverIO.Element) => {
```

Comment:
I was confused about `deleteText` name. I thought this function also deletes text for buttons.
Maybe need to rename it to `deleteInputText` or `clearInput` or something like that?

Reply:
`clearInput` sounds good 👍